config/configmap/inferenceservice.yaml

```diff
@@ -425,7 +425,21 @@ data:
          # ModelMesh https://kserve.github.io/website/master/admin/modelmesh/
          "defaultDeploymentMode": "Serverless"
        }
-     
+    
+     # ====================================== SERVICE CONFIGURATION ======================================
+     # Example
+     service: |-
```

Comment:
We had defined a new option in the config to not mix the configuration purpose, this affects only the service created, for now only Raw.
But in the future, more fields can be easily added here if needed..

Reply:
Is this ok @israel-hdez

Reply:
To me, the `deploy` is the key that should hold the config, as my thinking is that this is a config related to deploying models. I see first-level keys in this ConfigMap to be high-level fields, rather than low-level resource-focused ones.

But I think I prefer to defer to @yuzisun that decision.

Reply:
sorry for the late reply, I think we this is actually related to the ingress config section. The whole purpose to set ClusterIP to None is that you want to enable your LB right?

Reply:
@yuzisun, it might not be related to ingress, in this case, it is just to configure headless services.

> enable your LB right

Yes

Reply:
I don't have a strong opinion as well, both work for me. Initially, I did this way because, if for another reason more customizations are needed in the service object, this could be used.